src/common-lib/middleware/requestCounter.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    PeriodicExportingMetricReader, MeterProvider
+} from '@opentelemetry/sdk-metrics';
+import { OTLPMetricExporter } from '@opentelemetry/exporter-metrics-otlp-http';
+import { Resource } from '@opentelemetry/resources'
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions'
+
+const resource = Resource.default().merge(
+    new Resource({
+        [SemanticResourceAttributes.SERVICE_NAME]: 'my-app',
+        [SemanticResourceAttributes.SERVICE_VERSION]: '0.1.0',
+    })
+)
+
+const metricExporter = new OTLPMetricExporter({
```

Comment:
Create the exporter in the instrumentation so all the otel code in one place. In this middleware you just have to create the meter.